templates/sheets/item/profession-sheet.hbs

```diff
@@ -145,7 +157,7 @@
                              button=true editable=editable}}
                 </div>
                 <div class="profession-card third-path">
-                    <div class="flex third-path-header">
+                    <div class="flex third-path-header profession-display">
```

Comment:
if you want the profession-display everywhere you can put it on a surrounding element